### ISSUE TITLE
Exporting sample graph from homepage produces broken image

### ISSUE THREAD

**Issue by tonyhignett** [naver/billboard.js#2567]
## Description

I forked the codepen sample from https://naver.github.io/billboard.js/ and added some basic code to export it. The result is this broken image:

![billboard-chart](https://user-images.githubusercontent.com/2363023/155125836-eacabdea-f02d-4722-8c7e-cfef94d1dc8c.png)

## Steps to check or reproduce

Click the link in this codepen: https://codepen.io/tonyhignett/pen/JjOvmJR. The JS is just
```
bb.generate({
    bindto: "#chart",
    data: {
        columns: [
            ["data1", 30, 200, 100, 170, 150, 250],
            ["data2", 130, 100, 140, 35, 110, 50]
        ],
        types: {
          data1: "line",
          data2: "area-spline"
        },
        colors: {
          data1: "red",
          data2: "green"
        }
    }
}).export({mimeType: "image/png"}, dataUrl => {
    const link = document.createElement("a");

    link.download = `${Date.now()}.png`;
    link.href = dataUrl;
    link.innerHTML = "Download chart as image";

    document.body.appendChild(link);
 });
```

**Comment by netil** [comment]
Hi @tonyhignett, to export as image, it needs to access stylesheet to apply its styles.
Loading your codepen example, from the console you'll se below error log.

<img width="701" alt="image" src="https://user-images.githubusercontent.com/2178435/155462865-fe17e671-c43b-48ea-bc80-a1a13d324d13.png">

The `SecurityError` happens, because from the codepen.io, can't access the styles loaded from the different domain.
- https://cdn.jsdelivr.net/npm/billboard.js/dist/billboard.min.css

So, basically to make exported with correct styles applied, the style should be accessible from the current domain where you're triggering the `.export()`.

Just simply making all the styles to be loaded from the same domain, will work as expected.
- https://codepen.io/netil/pen/eYejJZe

https://naver.github.io/billboard.js/release/latest/doc/Chart.html#export
